pkg/api/testapi/testapi.go

```diff
@@ -279,6 +290,7 @@ func init() {
 	Rbac = Groups[rbac.GroupName]
 	Storage = Groups[storage.GroupName]
 	ImagePolicy = Groups[imagepolicy.GroupName]
+	Authorization = Groups[authorization.GroupName]
```

Comment:
If nobody uses this, don't add it.  We want less code to touch for each group you add, not more.